src/modules/home/components/SignInComponent.vue

```diff
@@ -66,11 +56,48 @@ export default defineComponent({
         selectedWallet.value != ''
       )
     );
+    
+     NetworkStateUtils.changeNetworkByIndex(1)
```

Comment:
should change to index 0

Reply:
@anakin001 , please take note. I think 1 is testnet1 is it @sleepyOwl14 ?

Reply:
ya, 1 is testnet1. but what happen to deployment with only 1 network ? there will be no index 1

Reply:
This is hardcoded as they decided to have only testnet1 for staging on the new design. Will update when we bring back select dropdown for network? is it correct @anakin001 ?